pkg/image/apis/image/types.go

```diff
@@ -469,6 +469,36 @@ type ImageStreamImage struct {
 // DockerImageReference points to a Docker image.
 type DockerImageReference = reference.DockerImageReference
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageStreamLayers describes information about the layers referenced by images in this
+// image stream.
+type ImageStreamLayers struct {
+	metav1.TypeMeta
+	// Standard object's metadata.
+	metav1.ObjectMeta
+	// blobs is a map of blob name to metadata about the blob.
+	Blobs map[string]ImageLayerData
+	// layers is a map between an image name and the names of the blobs that form the
```

Comment:
just to confirm, the key here is the `.image` element from the `.status.tags[key].image`?